release-notes.txt

```diff
@@ -0,0 +1,208 @@
+BISDN Linux
+v4.6.0 Release Notes
+Date: 2022-05-03
+
+! New in this release
```

Comment:
We have a new OF-DPA option for controlling memory allocation for Layer 2 and Layer 3 tables which should be mentioned here.

Also it would be good to thematically group things, so have OF-DPA things together.